### PR TITLE
Fix state proof check failure on failed transaction check

### LINKED ISSUE #1257: Monitor state proof check failure
**Detailed Description**

The new State Proof Alpha monitor checks added to the REST monitor fails when checking for failed transactions. This is because the URL it uses to return the failed transaction actually contains both a failed and a successful transaction. Both are for the same transaction ID submitted to different nodes, hence only one was successful and the other marked as DUPLICATE_TRANSACTION.

URL it checked:
`http://testnet.mirrornode.hedera.com/api/v1/transactions/0.0.47439-1605137702-479000000/stateproof`
```json
{
  "transactions":[
     {"consensus_timestamp":"1605137714.150262002","transaction_hash":"+waNk7qYWRjFICZxAxxCiyZ4sc0W5ZPpswMPTowc5oRuTUQX2x+1K4CX4b3WWWiX","valid_start_timestamp":"1605137702.479000000","charged_tx_fee":507341,"memo_base64":"","result":"SUCCESS","name":"CONSENSUSSUBMITMESSAGE","max_fee":"200000000","valid_duration_seconds":"120","node":"0.0.5","transaction_id":"0.0.47439-1605137702-479000000","transfers":[{"account":"0.0.5","amount":33625},{"account":"0.0.98","amount":473716},{"account":"0.0.47439","amount":-507341}]},
     {"consensus_timestamp":"1605137714.180729001","transaction_hash":"AoxqNJ5+H5fhz9ylTgI9q1h1DfIzn96JWkE2xSkjXnruM9NoE5GodcnSpJ35k7X/","valid_start_timestamp":"1605137702.479000000","charged_tx_fee":470528,"memo_base64":"","result":"DUPLICATE_TRANSACTION","name":"CONSENSUSSUBMITMESSAGE","max_fee":"200000000","valid_duration_seconds":"120","node":"0.0.6","transaction_id":"0.0.47439-1605137702-479000000","transfers":[{"account":"0.0.6","amount":33625},{"account":"0.0.98","amount":436903},{"account":"0.0.47439","amount":-470528}]}
  ]
}
```

**Actual Behavior**
Monitor shows `expect http error 404, got 2xx`.

**Expected Behavior**
All checks pass

**Environment:**
- OS: Ubuntu 18.04
- Version: 0.22.0

**Additional Context**
Add any other context about the problem here. Attach any logs here, if applicable.

### PR DESCRIPTION
**Pull request by xin-hedera**
**Detailed description**:
Remove all negative stateproof tests

 
**Which issue(s) this PR fixes**:
Fixes #1257 

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: hedera-mirror-rest/monitoring/monitor_apis/stateproof_tests.js [hunk 1]
```diff
@@ -90,22 +93,27 @@ const checkStateproofForValidTransaction = async (server) => {
  * @return {{url: String, passed: boolean, message: String}}
  */
 const checkStateproofForFailedTransaction = async (server) => {
-  let url = getUrl(server, transactionsPath, {limit: 1, order: 'desc', result: 'fail'});
+  let url = getUrl(server, transactionsPath, {limit: failedTransactionLimit, order: 'desc', result: 'fail'});
```
Comment (steven-sheehy) [inline hedera-mirror-rest/monitoring/monitor_apis/stateproof_tests.js:96]: It doesn't look like any of these query params are applicable to `/transactions/:transaction_id`.

File: hedera-mirror-rest/monitoring/README.md [hunk 2]
```diff
@@ -63,6 +63,8 @@ To customize per-resource configuration:
 - set `token.tokenId` to an existing token ID of the target environment in case it's needed to pass certain token tests,
   e.g., `tokenBalancesLimit` for the token. If not set, the tests will use the first token from the
   token discovery API response
+- set `failedTransactionLimit` for `stateproof` to limit the number of failed transactions to search for a non
```
Comment (steven-sheehy) [inline hedera-mirror-rest/monitoring/README.md:66]: This seems like a hack. When we have more than limit nodes in the future someone might submit to all and this fail again.

**Review by steven-sheehy — CHANGES_REQUESTED** [review]
I think this check (and the not exist check) is unlike the other checks in that it is a negative test instead of a positive test. I think negative checks are probably more appropriate for unit/functional tests and might not belong in a monitor. 

I think to solve this issue we can just remove these two calls from the monitor.

File: hedera-mirror-rest/monitoring/monitor_apis/stateproof_tests.js (hunk 1 above)
Comment (xin-hedera) [reply hedera-mirror-rest/monitoring/monitor_apis/stateproof_tests.js]: transactionsPath is just `/transactions`, without `/:transactionId`

**Review by ijungmann — APPROVED** [review]
LGTM

**Review by steven-sheehy — APPROVED** [review]

**Pull request merged**